docs/topics/components-stability.md

```diff
@@ -45,21 +45,23 @@ kotlin-stdlib (JVM)|Stable|1.0| |
 Coroutines|Stable|1.3| |
 kotlin-reflect (JVM)|Beta|1.0| |
 Kotlin/JS (Classic back-end)|Stable|1.3| |
-Kotlin/JVM (IR-based)|Alpha|1.4| |
+Kotlin/JVM (IR-based)|Stable|1.5| |
 Kotlin/JS (IR-based)|Beta|1.5| |
 Kotlin/Native Runtime|Beta|1.3| |
 KLib binaries|Alpha|1.4| |
 KDoc syntax|Stable|1.0| |
-dokka|Alpha|0.1| |
-Kotlin Scripts (*.kts)|Beta|1.2| |
-Kotlin Scripting APIs and custom hosts|Alpha|1.2| |
+Dokka|Alpha|0.1| |
+Scripting syntax and semantics|Alpha|1.2|
+Scripting embedding and extension API|Beta|1.5
+Scripting IDE support|Experimental|1.2
```

Comment:
Just curious: why decided to divide into four different lines about scripting?

Reply:
The team suggested a new component division with a new status line-up that better reflects the current state of the scripting feature